components/templates/src/global_fns/files.rs

```diff
@@ -399,6 +399,29 @@ title = "A title"
         );
     }
 
+    #[test]
+    fn does_not_duplicate_lang() {
```

Comment:
what does it mean to duplicate lang?

Reply:
It's described in https://github.com/getzola/zola/issues/1878.

TL;DR: currenly `get_url(path='/lang/something', lang='lang')` will return a URL with duplicated `lang` part: `/lang//lang/something`